components/textfield/index.css

```diff
@@ -131,207 +458,233 @@ governing permissions and limitations under the License.
   */
   -moz-appearance: textfield;
 
-  &::placeholder {
-    font-weight: var(--spectrum-textfield-texticon-placeholder-text-font-weight);
-    font-style: var(--spectrum-textfield-texticon-placeholder-text-font-style);
-    transition: color var(--spectrum-global-animation-duration-100) ease-in-out;
-    opacity: 1;
+  /* Remove the native clear button in IE */
+  /* http://stackoverflow.com/questions/14007655/remove-ie10s-clear-field-x-button-on-certain-inputs */
+  &::-ms-clear {
+    inline-size: 0;
+    block-size: 0;
+  }
+
+  /* removes the native spin buttons */
+  /* http://stackoverflow.com/questions/23372903/hide-spinner-in-input-number-firefox-29 */
+  &::-webkit-inner-spin-button,
+  &::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+
+  /* removes the red border that appears in Firefox */
+  &:-moz-ui-invalid {
+    box-shadow: none;
   }
 
   /* added to work with Edge, note, it needs double ::
    * not single : which is what autoprefixer will add
    */
   &::-ms-input-placeholder {
-    font-weight: var(--spectrum-textfield-texticon-placeholder-text-font-weight);
-    font-style: var(--spectrum-textfield-texticon-placeholder-text-font-style);
-    transition: color var(--spectrum-global-animation-duration-100) ease-in-out;
     opacity: 1;
   }
 
+  /*** Input Placeholder Text ***/
+  &::placeholder {
+    opacity: 1;
+    font-size: var(--mod-textfield-placeholder-font-size, var(--spectrum-textfield-placeholder-font-size));
+    font-family: var(--mod-textfield-font-family, var(--spectrum-textfield-font-family));
+    font-weight: var(--mod-textfield-font-weight, var(--spectrum-textfield-font-weight));
+    color: var(--highcontrast-textfield-text-color-default, var(--mod-textfield-text-color-default, var(--spectrum-textfield-text-color-default)));
+    transition: color var(--mod-animation-duration-100, var(--spectrum-animation-duration-100)) ease-in-out;
```

Comment:
Should a component alias be defined for this global `spectrum-animation-duration`?